roles/confluent.kafka_connect/tasks/connectors.yml

```diff
@@ -7,7 +7,7 @@
     owner: "{{kafka_connect_user}}"
     mode: '764'
   when: item != '/usr/share/java'
-  with_items: "{{ kafka_connect.properties['plugin.path'].split(',') }}"
+  with_items: "{{ kafka_connect_final_properties['plugin.path'].split(',') }}"
```

Comment:
Might be worth leaving a comment that confirms where the properties are merged together and then pull from. Can be a a bit confusing to see an almost empty properties file.